packages/harmony/src/components/radio-group/useRadioGroup.ts

```diff
@@ -0,0 +1,35 @@
+import { ChangeEventHandler, useCallback, useContext } from 'react'
+
+import { RadioGroupContext } from './RadioGroupContext'
+
+type RadioGroupProps = {
+  name?: string
+  checked?: boolean
+  value?: any
+  onChange?: ChangeEventHandler<HTMLInputElement>
+}
+
+export const useRadioGroup = (props: RadioGroupProps) => {
```

Comment:
this was written in multiple places, so simply consolidates. what it does is check if there is a wrapping radio context, and if so ensure radio-group's onChange is fired